src/main/java/ua/softserveinc/tc/util/DateUtil.java

```diff
@@ -41,6 +43,10 @@ public static Date toDate(String date) {
         }
     }
 
+    public static Date toDate(LocalDate date) {
```

Comment:
Please don't read this blog any more. It have completely bad explanations and misleading code. As you can see in the bottom the result time will be -> 00:00:00.